src/bindings/lua/wreck.lua

```diff
@@ -289,6 +290,11 @@ function wreck:parse_cmdline (arg)
         self.ncores = self.ntasks
     end
 
+    self.ngpus = 0
```

Comment:
I don't think this line is necessary due to 
```lua
ngpus = self.ngpus or 0
```
below. However, it doesn't hurt so it can stay for now.